docs/pages/eas-update/rollouts.mdx

```diff
@@ -3,6 +3,25 @@ title: Rollouts
 hideTOC: true
 ---
 
-EAS Update does not support percentage-based rollouts. With that said, we built EAS Update with percentage-based rollouts in mind. We plan to work on rollouts in 2023. With this feature, you'll be able to incrementally roll out one update over another to a percentage of your end-users.
+Rollouts allow us to deploy a new branch to a specific channel incrementally. With rollouts, we can roll out Branch B over Branch A by percentage. This is useful when we want to test a new feature to minimize the risk of introducing bugs or other issues to our production environment.
```

Comment:
```suggestion
import { Terminal } from '~/ui/components/Snippet';

Rollouts allow deploying a new branch to a specific channel incrementally. With rollouts, you can roll out Branch B over Branch A by percentage. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to your production environment.
```

- [Since this is not a tutorial, as per our writing styleguide, let's use second person voice for the whole document.](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#write-in-the-second-person)
- I'm not sure I understand the context "Branch B over Branch A by percentage." Confusing part is "by percentage". This also doesn't general tell general audience about difference between two branches. I think you might want to say, "...one branch over the other" or something like that.
- Please import the `<Terminal>` component as it'll be used in the content below for bash scripts.